### PR TITLE
Feature/docker compose - mongodb

### PR DESCRIPTION
**Pull request by omerzamir**
added mongodb to the Docker-Compose file.

#2 [link to issue](#2 )

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,4 +5,19 @@ services:
...
+    environment:
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+    volumes:
+      - mongodb_data:/var/lib/mongodb
+    ports:
+      - 27017:27017
+
+volumes:
+  mongodb_data:
```
Comment (omrishtam) [inline docker-compose.yml:24]: What is mongodb_data volume?

File: docker-compose.yml [hunk 2]
```diff
@@ -5,4 +5,19 @@ services:
     image: php-server
```
Comment (omrishtam) [inline docker-compose.yml:5]: php-server should eventually depend on mongodb so i think you should add that to the php-server service

**Review by omrishtam — COMMENTED** [review]
I think you should also update the docker-compose.debug.yml file and put its volume to point to ~/Desktop/mongodb or something more comfortable than /var/lib/mongodb.
Also i think the mongodb volume in docker-compose.yml should point to /data/db aswell, /var/lib/mongodb is the db location of old versions of mongodb (like v2.4...).

File: docker-compose.yml [hunk 3]
```diff
@@ -5,4 +5,19 @@ services:
...
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+    volumes:
+      - mongodb_data:/var/lib/mongodb
+    ports:
+      - 27017:27017
+
+volumes:
+  mongodb_data:
+    driver: local
```
Comment (omrishtam) [inline docker-compose.yml]: missing newline

File: docker-compose.yml (hunk 1 above)
Comment (omerzamir) [reply docker-compose.yml:24]: this is the volume we want to map do the mongodb service

Comment (omrishtam) [reply docker-compose.yml:24]: But what location is it in the container? And you set MONGO_DATA_DIR=/data/db and there's no use to that volume.

**Review by omrishtam — COMMENTED** [review]
LGTM

**Review by omrishtam — COMMENTED** [review]
LGTM

**Pull request merged**